docs/DeveloperGuide.md

```diff
@@ -306,68 +315,81 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  User provides client details to add
-2.  Clientele+ adds the client
+1.  User provides client details to add.
+2.  Clientele+ adds the client.
 
     Use case ends.
 
 **Extensions**
 
-* 1a. The client's name is not provided
+* 1a. The client's name, phone, email, address, or deadline is not provided.
 
-   * 1a1. Clientele+ shows an error message
-   * 1a2. User reenters new command
-   Steps 1a1-1a2 are repeated until the command entered is correct
+   * 1a1. Clientele+ shows an error message.
+   * 1a2. User reenters new command.
+   
+        Steps 1a1-1a2 are repeated until the command entered is correct.
 
-  Use case resumes at step 1.
+    Use case resumes at step 1.
 
-* 1b. Input fields are incorrectly formatted
+* 1b. Input fields are incorrectly formatted.
 
     * 1b1. Clientele+ shows an error message.
-    * 1b2. User reenters command
-    Steps 1b1-1b2 are repeated until the command entered is correct
+    * 1b2. User reenters command. 
+
+      Steps 1b1-1b2 are repeated until the command entered is correct.
 
```

Comment:
By any chance, would it be good to add the situation where at any time they decide to stop adding the client / exit the app?